uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/fragment/QuotesFragment.kt

```diff
@@ -96,6 +99,7 @@ class QuotesFragment : Fragment(), QuotesSortView.Listener,
     ): View? {
         val view = inflater.inflate(R.layout.uisdk_quotes_fragment, container, false)
 
+        progressBarWidget = view.findViewById(R.id.progressBar)
```

Comment:
this needs to be hidden in the `onCreate` method. otherwise then you go back from the checkout the progress bar is shown even though there is nothing loading. when necessary, the changedStatus method will kick in to change the visibility